modules/prelude-org.el

```diff
@@ -58,6 +58,42 @@
 
 (add-hook 'org-mode-hook (lambda () (run-hooks 'prelude-org-mode-hook)))
 
+(defun prelude-enable-org-mode-shift-bindings ()
```

Comment:
Won't it be much easier to just disable the `windmove` stuff in the `prelude-org-mode-hook`?